### PR TITLE
Fix user guide

### PR DESCRIPTION
**Pull request by rionshocker**

### CODE REVIEW

**Review by Khaleelur-Rahman — APPROVED** [review]
LGTM!

File: docs/UserGuide.md [hunk 1]
```diff
@@ -188,20 +188,19 @@ Format: `find_contact KEYWORD [MORE_KEYWORDS]`
 Examples:
 * `find_contact John` returns `john` and `John Doe`.
 * `find_contact alex david` returns `Alex Yeoh`, `David Li`.
+* `find_contacts doe` after `find_contacts john` will return `John Doe` and not `john` who was originally in the result.
```
Comment (vijay-shankaranand) [inline docs/UserGuide.md]: should this be find_contact?

Comment (Khaleelur-Rahman) [inline docs/UserGuide.md:191]: `find_contact` and not `find_contacts`

Comment (rionshocker) [reply docs/UserGuide.md]: Good catch! Have changed it.

**Review by vijay-shankaranand — APPROVED** [review]
LGTM besides minor nit

File: docs/UserGuide.md (hunk 1 above)
Comment (rionshocker) [reply docs/UserGuide.md]: Yup. Have changed accordingly, thanks!

**Pull request merged**